<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="action_Export_as_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>F&amp;ormat</string>
    </property>
    <widget class="QMenu" name="menuCharacter">
     <property name="title">
      <string>Character</string>
     </property>
     <addaction name="actionFont"/>
     <addaction name="actionBold"/>
     <addaction name="actionItalic"/>
     <addaction name="actionUnderline"/>
    </widget>
    <addaction name="menuCharacter"/>
    <addaction name="separator"/>
    <addaction name="actionLeft"/>
    <addaction name="actionRight"/>
    <addaction name="actionCenter"/>
    <addaction name="actionJustify"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
    <addaction name="actionHighlighter"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Insert">
    <property name="title">
     <string>&amp;Insert</string>
    </property>
    <addaction name="actionImage"/>
    <addaction name="actionTable"/>
    <addaction name="actionList"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Insert"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNewWinodw">
   <property name="text">
    <string>&amp;New Winodw</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>&amp;Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>&amp;Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="text">
    <string>&amp;Underline</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>&amp;Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>&amp;Right</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="text">
    <string>&amp;Center</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="text">
    <string>&amp;Justify</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>&amp;Color</string>
   </property>
  </action>
  <action name="action_Export_as_PDF">
   <property name="text">
    <string>&amp;Export as PDF...</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>&amp;Image</string>
   </property>
   <property name="toolTip">
    <string>Image</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>&amp;Table</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>&amp;List</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHighlighter">
   <property name="text">
    <string>&amp;Highlighter</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>&amp;Font</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="text">
    <string>&amp;Print Preview...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
